.gitignore

```diff
@@ -1,2 +1,5 @@
 *.pyc
 temp/
+cdpedia-es*
+cdpetron.log
+.idea
```

Comment:
Acá github me está botoneando que no hay una línea en blanco al final del archivo. Debería agregarla?

Reply:
Si, agregala.
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Reply:
Gracias, ahora si quedó.